library/core/class.bbcode.php

```diff
@@ -311,6 +311,9 @@ public function format($bbcode) {
     public function media() {
         if ($this->media === null) {
             $controller = Gdn::controller();
+            if (!$controller instanceof Gdn_Controller) {
```

Comment:
```suggestion
            if ((!$controller instanceof Gdn_Controller)) {
```

Without this it never runs (Essentially the same as `false instanceof Gdn_Controller`).